fluent-terminal.json

```diff
@@ -0,0 +1,21 @@
+{
+    "homepage": "https://github.com/felixse/FluentTerminal",
+    "description": "A Terminal Emulator based on UWP and web technologies.",
+    "license": {
+        "identifier": "GNU General Public License v3.0",
+        "url": "https://raw.githubusercontent.com/felixse/FluentTerminal/master/LICENSE"
+    },
+    "version": "0.6.0", 
+    "url": "https://github.com/felixse/FluentTerminal/releases/download/0.6.0.0/FluentTerminal.Package_0.6.0.0_Test.zip",
+    "hash": "942c5d55a4183942a00b7cf8c3309dd1092402f5c97a479e7d6b4c0309167ae3",
+    "pre_install": [
+        "$thumb='8A5DE5B20B663BD6A99B44C5BA278289521F419F';",
+        "$store='TrustedPeople';",
+        "if (Get-ChildItem -Path Cert:\\LocalMachine\\$store | Where-Object {$_.Thumbprint -eq $thumb}) { return }",
+        "sudo certutil -addstore $store .\\FluentTerminal.Package_0.6.0.0_x86_x64.cer; gpupdate \/force;",
+        "sudo reg add 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\AppModelUnlock' \/t REG_DWORD \/f \/v 'AllowAllTrustedApps' \/d '1';"
+    ],
+    "installer": {
+        "script": ".\\Install.ps1 -ForceContextMenu -Force"
```

Comment:
```suggestion
        "script": "& \"$dir\\Install.ps1\" -ForceContextMenu -Force"
```

Reply:
Why `script` instead of `file` and `args`?